programs/dummy-axelar-solana-gateway/src/instructions.rs

```diff
@@ -3,7 +3,13 @@
 use std::fmt::Debug;
 
 use borsh::{to_vec, BorshDeserialize, BorshSerialize};
-use solana_program::{instruction::Instruction, pubkey::Pubkey};
+use solana_program::{
+    instruction::{AccountMeta, Instruction},
+    pubkey::Pubkey,
```

Comment:
Well this program was called `dummy-axelar-solana-gateway`, but i think it hsould be rename to be `dummy-axelar-solana-program` as we use it to mock or test scenarios that require logic to be executed inside a program.

Reply:
created https://github.com/eigerco/solana-axelar/issues/120 to address it later